content/docs/command-reference/remote/add.md

```diff
@@ -125,34 +125,40 @@ So, make sure you have the following permissions enabled:
 
 <details>
 
-### Click for S3 API compatible storage
+### Click for S3-compatible storage
 
-To communicate with a remote object storage that supports an S3 compatible API
+To communicate with a remote object storage that supports an S3-compatible API
 (e.g. [Minio](https://min.io/),
 [DigitalOcean Spaces](https://www.digitalocean.com/products/spaces/),
 [IBM Cloud Object Storage](https://www.ibm.com/cloud/object-storage) etc.),
-configure the remote's `endpointurl` explicitly:
-
-For example:
+configure the remote's `endpointurl`:
 
 ```dvc
 $ dvc remote add -d myremote s3://mybucket/path
 $ dvc remote modify myremote endpointurl \
                     https://object-storage.example.com
 ```
 
-> See `dvc remote modify` for a full list of S3 API parameters.
+It's also important to setup appropriate authentication with any of the related
+parameters available for regular S3 storage (see `dvc remote modify`).
+Otherwise, DVC may try to use default AWS credentials (if any), which will
+probably cause an error. Example:
```

Comment:
Resubmitting https://github.com/iterative/dvc.org/pull/2415#discussion_r619635067:
```suggestion
⚠️ It's important to setup appropriate authentication with `dvc remote modify`.
Otherwise, DVC may use default AWS credentials, which would cause an error.
```
And now I just merge it directly in the PR. Cc @iesahin